src/scss/lib/texts.scss

```diff
@@ -26,6 +26,12 @@
   }
 }
 
+.subtitle-large{
+  font-size: 18px;
+  font-weight: 400;
+  line-height: 21px;
+}
+
 .subtitle-link{
   font-family: $font-menu,sans-serif;
   color: #ffffff;
```

Comment:
We should use variables for the main colors hex codes.

Reply:
Fixed